### PR TITLE
Adds the password policy step to the scenario

### PR DESCRIPTION
**Pull request by burtlo**
Associated with the [Dynamic Secrets: Database Secrets](https://learn.hashicorp.com/tutorials/vault/database-secrets#prerequisites) tutorial.
[Asana Task](https://app.asana.com/0/563192436488770/1199102547267053)

- Adds the step to create and read a password policy
- Updates the admin policy to enable policy management

### CODE REVIEW

File: vault-dynamic-secrets/step6.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+The database secret engines generate passwords that adhere to a default pattern
+that may be overriden with a new password policy. A policy defines the rules and
```
Comment (brianshumate) [inline vault-dynamic-secrets/step6.md:2]:
```suggestion
that may be overridden with a new password policy. A policy defines the rules and
```

**Review by brianshumate — CHANGES_REQUESTED** [review]
Looks great to me! I found only just one little typo.

**Review by yhyakuna — APPROVED** [review]
LGTM

**Review by topfunky — APPROVED** [review]
👍🏻

**Pull request merged**